app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitListFragment.java

```diff
@@ -200,6 +204,7 @@ private void initializeRecyclerView(LinearLayoutManager layoutManager, View view
             public void onRowClicked(int position) {
                 // Get the selected habit
                 Habit currentHabit = _habitItemAdapter._snapshots.get(position);
+                Log.d(TAG, "onRowClicked: current habit:" + currentHabit.getPublic());
```

Comment:
Minor: this log message can be improved to `onRowClicked: is current habit public:`. If we don't have any other context, `current habit: true` doesn't really make sense.

Reply:
Removed the line, only used it to troubleshoot an issue and that issue was solved.